site/docs/patterns/button-bar.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Button Bar
+layout: DetailPattern
+aliases:
+  - /salt/patterns/button-bar
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-button-bar--button-bar", label: "Examples" },
+    ]
+---
+
+Button Bar is a pattern that displays possible actions for users to take when they reach the end of a task. You can use it to provide consistent design when implementing actionable interactions within pages or dialogs across your applications.
+
+  <Diagram
+    src="/img/patterns/button-bar/with-secondary.png"
+    alt="Example button bar, with CTA to the left then primary, on the right side there are two secondary buttons"
+    contentPosition="top"
+    border
+  />
+
+We provide design guidance of how you can use Salt Button and Layout components and Button Bar template to achieve the pattern. Implementation examples can also be seen in the storybook.
```

Comment:
would it be beneficial to link to Salt components whenever they're mentioned in this pattern

Reply:
yeah I think for the initial mention it makes sense. Although, Layout isn't something we can link to directly so would have to mention all the layouts used